backend/hct_mis_api/apps/targeting/tests/test_admin.py

```diff
@@ -0,0 +1,26 @@
+from hct_mis_api.apps.household.tests.test_admin import BaseTest
+from hct_mis_api.apps.targeting.fixtures import TargetPopulationFactory
+
+
+# temporary added test for AutoCompleteFilterTemp have to be removed after fix in AutoCompleteFilter
+class HouseholdSelectionAdminTest(BaseTest):
+    def test_household_selection_filtering_by_targeting(self) -> None:
+        base_url = "/api/unicorn/targeting/householdselection/"
+        targeting_removed = TargetPopulationFactory(name="Test TP for test", is_removed=True)
+        targeting_1 = TargetPopulationFactory(name="Test TP for test 222")
+        self.assertTrue(targeting_removed.is_removed)
+        self.assertFalse(targeting_1.is_removed)
+
+        # removed tp
+        url = f"{base_url}?&target_population__exact={str(targeting_removed.id)}"
+        res = self.app.get(url, user=self.superuser)  # noqa: F841
```

Comment:
why noqa? F841 is `Local variable name is assigned to but never used`, it doesn't look like this case

Reply:
my bad, just copy-pasted from somewhere. updated 👌